The files grub-mkconfig_lib and update-grub_lib are installed into
$libdir/@PACKAGE@.

=== modified file 'util/grub-mkconfig.in'
--- util/grub-mkconfig.in	2010-11-13 15:27:29 +0000
+++ util/grub-mkconfig.in	2011-01-08 23:35:39 +0000
@@ -95,7 +95,7 @@ do
     esac
 done
 
-. ${libdir}/grub/grub-mkconfig_lib
+. ${libdir}/@PACKAGE@/grub-mkconfig_lib
 
 if [ "x$EUID" = "x" ] ; then
   EUID=`id -u`

=== modified file 'util/grub.d/00_header.in'
--- util/grub.d/00_header.in	2010-12-10 11:45:08 +0000
+++ util/grub.d/00_header.in	2011-01-08 23:35:39 +0000
@@ -25,7 +25,7 @@ libdir=@libdir@
 locale_dir=`echo ${GRUB_PREFIX}/locale | sed ${transform}`
 grub_lang=`echo $LANG | cut -d . -f 1`
 
-. ${libdir}/grub/grub-mkconfig_lib
+. ${libdir}/@PACKAGE@/grub-mkconfig_lib
 
 # Do this as early as possible, since other commands might depend on it.
 # (e.g. the `loadfont' command might need lvm or raid modules)

=== modified file 'util/grub.d/10_hurd.in'
--- util/grub.d/10_hurd.in	2010-12-21 12:49:29 +0000
+++ util/grub.d/10_hurd.in	2011-01-08 23:35:39 +0000
@@ -20,7 +20,7 @@ set -e
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
-. ${libdir}/grub/grub-mkconfig_lib
+. ${libdir}/@PACKAGE@/grub-mkconfig_lib
 
 CLASS="--class gnu --class os"
 

=== modified file 'util/grub.d/10_kfreebsd.in'
--- util/grub.d/10_kfreebsd.in	2010-12-21 12:49:29 +0000
+++ util/grub.d/10_kfreebsd.in	2011-01-08 23:35:39 +0000
@@ -21,7 +21,7 @@ prefix=@prefix@
 exec_prefix=@exec_prefix@
 bindir=@bindir@
 libdir=@libdir@
-. ${libdir}/grub/grub-mkconfig_lib
+. ${libdir}/@PACKAGE@/grub-mkconfig_lib
 
 export TEXTDOMAIN=@PACKAGE@
 export TEXTDOMAINDIR=@localedir@

=== modified file 'util/grub.d/10_linux.in'
--- util/grub.d/10_linux.in	2010-12-21 12:49:29 +0000
+++ util/grub.d/10_linux.in	2011-01-08 23:35:39 +0000
@@ -21,7 +21,7 @@ prefix=@prefix@
 exec_prefix=@exec_prefix@
 bindir=@bindir@
 libdir=@libdir@
-. ${libdir}/grub/grub-mkconfig_lib
+. ${libdir}/@PACKAGE@/grub-mkconfig_lib
 
 export TEXTDOMAIN=@PACKAGE@
 export TEXTDOMAINDIR=@localedir@

=== modified file 'util/grub.d/10_netbsd.in'
--- util/grub.d/10_netbsd.in	2010-10-12 12:47:04 +0000
+++ util/grub.d/10_netbsd.in	2011-01-08 23:35:39 +0000
@@ -21,7 +21,7 @@ prefix=@prefix@
 exec_prefix=@exec_prefix@
 bindir=@bindir@
 libdir=@libdir@
-. ${libdir}/grub/grub-mkconfig_lib
+. ${libdir}/@PACKAGE@/grub-mkconfig_lib
 
 export TEXTDOMAIN=@PACKAGE@
 export TEXTDOMAINDIR=@localedir@

=== modified file 'util/grub.d/10_windows.in'
--- util/grub.d/10_windows.in	2010-06-29 15:20:49 +0000
+++ util/grub.d/10_windows.in	2011-01-08 23:35:39 +0000
@@ -20,7 +20,7 @@ set -e
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
-. ${libdir}/grub/grub-mkconfig_lib
+. ${libdir}/@PACKAGE@/grub-mkconfig_lib
 
 case "`uname 2>/dev/null`" in
   CYGWIN*)  ;;

=== modified file 'util/grub.d/20_linux_xen.in'
--- util/grub.d/20_linux_xen.in	2010-12-31 22:49:42 +0000
+++ util/grub.d/20_linux_xen.in	2011-01-08 23:35:39 +0000
@@ -21,7 +21,7 @@ prefix=@prefix@
 exec_prefix=@exec_prefix@
 bindir=@bindir@
 libdir=@libdir@
-. ${libdir}/grub/grub-mkconfig_lib
+. ${libdir}/@PACKAGE@/grub-mkconfig_lib
 
 export TEXTDOMAIN=@PACKAGE@
 export TEXTDOMAINDIR=@localedir@

=== modified file 'util/grub.d/30_os-prober.in'
--- util/grub.d/30_os-prober.in	2010-12-25 17:16:01 +0000
+++ util/grub.d/30_os-prober.in	2011-01-08 23:35:39 +0000
@@ -21,7 +21,7 @@ prefix=@prefix@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
 
-. ${libdir}/grub/grub-mkconfig_lib
+. ${libdir}/@PACKAGE@/grub-mkconfig_lib
 
 if [ "x${GRUB_DISABLE_OS_PROBER}" = "xtrue" ]; then
   exit 0

=== modified file 'util/update-grub_lib.in'
--- util/update-grub_lib.in	2008-11-20 19:22:20 +0000
+++ util/update-grub_lib.in	2011-01-08 23:35:39 +0000
@@ -18,6 +18,6 @@ prefix=@prefix@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
 
-. ${libdir}/grub/grub-mkconfig_lib
+. ${libdir}/@PACKAGE@/grub-mkconfig_lib
 
 grub_warn "update-grub_lib is deprecated, use grub-mkconfig_lib instead"

